CODEOWNERS

```diff
@@ -1 +1 @@
-sunsplat
+sunsplat,amelia2802,hernandezforges
```

Comment:
this will hopefully auto-notify you two so we don't have to manually request reviews with every PR